odoo/tools/misc.py

```diff
@@ -1640,9 +1640,10 @@ def hmac(env, scope, message, hash_function=hashlib.sha256):
     ).hexdigest()
 
 
-ADDRESS_REGEX = re.compile(r'^(.*?)(\s[0-9][0-9\S]*)?( - (.+))?$')
+ADDRESS_REGEX = re.compile(r'^(.*?)(\s[0-9][0-9\S]*)?( - (.+))?$', re.S)
```

Comment:
Use the full flag name, for clarity. Passing `flags` by kwarg is also a good habit to take, as some of the other regex methods have odd signatures which are easy to misuse when used to passing flags positionally e.g. `re.sub` and `re.split`.

Reply:
Also if group 2 is unused, it could be made non-capturing, also for clarity.

Reply:
`flags` keyword argument added; 
full flag name used;
non-capturing `?:` added to group[2]